sqlserver/assets/configuration/spec.yaml

```diff
@@ -713,6 +713,28 @@ files:
         type: number
         example: 1800
         display_default: false
+    - name: schemas_collection 
+      description: |
+        Configure collection of schemas.
```

Comment:
Do we want to mention here that users should enable db autodiscovery or set the `database` field? Otherwise, we will only collect schemas for the master db?

Reply:
sure, I'll add.